packages/manager/apps/web/client/app/hosting/boost/hosting-boost.service.js

```diff
@@ -35,6 +35,16 @@ angular
       return $q.all(tasks);
     };
 
+    /**
+     * Obtain hosting boost new price
+     */
+    this.getBoostPrice = ovhSubsidiary => OvhHttp.get('/order/catalog/public/webHosting?ovhSubsidiary={ovhSubsidiary}', {
```

Comment:
we tend to avoid `OvhHttp` use you should prefer using methods from https://github.com/ovh-ux/ovh-api-services
```suggestion
    this.getBoostPrice = ovhSubsidiary => OvhApiOrderCatalogPublic.v6().get({
    ovhSubsidiary,
    productName: 'hosting'
    }).$promise
```